include/ccf/endpoints/authentication/authentication_types.h

```diff
@@ -29,6 +29,8 @@ namespace ccf
   public:
     virtual ~AuthnPolicy() = default;
 
+    static constexpr auto SECURITY_SCHEME_NAME = "";
```

Comment:
```suggestion
    static constexpr auto SECURITY_SCHEME_NAME = "BaseAuthPolicy";
```

We should insert a default value here, so we can spot if it ever leaks through.

Reply:
Done